plaso/parsers/manager.py

```diff
@@ -45,6 +45,58 @@ def CreateSignatureScanner(cls, specification_store):
 
     return scanner_object
 
+  @classmethod
+  def CheckFilterExpression(cls, parser_filter_expression):
+    """Checks parser and plugin names in a parser filter expression.
+
+    Args:
+      parser_filter_expression (str): parser filter expression,
+          where None represents all parsers and plugins.
+
+          A parser filter expression is a comma separated value string that
+          denotes which parsers and plugins should be used. See
+          filters/parser_filter.py for details of the expression syntax.
+
+          This function does not support presets, and requires a parser filter
+          expression where presets have been expanded.
+
+    Returns:
+      tuple: containing:
+
+      * set(str): parser filter expression elements that contain known parser
+          and/or plugin names.
+      * set(str): parser filter expression elements that contain unknown parser
+          and/or plugin names.
+    """
+    if not parser_filter_expression:
+      return set(cls._parser_classes.keys()), set()
+
+    known_parser_elements = set()
+    unknown_parser_elements = set()
+    for element in parser_filter_expression.split(','):
+      parser_expression = element
+      if element.startswith('!'):
+        parser_expression = element[1:]
+
+      parser_name, _, plugin_name = parser_expression.partition('/')
```

Comment:
Why not use SplitExpression as in GetParserObjects, so that the parser filter logic is contained in ParserFilterExpressionHelper?

Reply:
I did consider that, but it makes it more difficult to display the element that cause the problem. This method would have to reconstruct the expression element, so the logic would still be here.